.github/workflows/coding-standards.yml

```diff
@@ -24,4 +24,4 @@ on:
 
 jobs:
   coding-standards:
-    uses: "doctrine/.github/.github/workflows/coding-standards.yml@2.1.0"
+    uses: "doctrine/.github/.github/workflows/coding-standards.yml@main"
```

Comment:
TODO: Change to `3.0.0` as soon as the tag has been created.

Reply:
Done